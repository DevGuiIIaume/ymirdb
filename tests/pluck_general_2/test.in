set a 1
set b a
set c b
set d a b c
list entries
pluck d 1
pluck d 2
pluck d 2
pluck d 1
list entries
forward d
backward d
forward a
backward a
forward b
backward d
forward c
backward c
push d a b c
list entries
forward a
forward b
forward c
forward d
backward a
backward b
backward c
backward d
bye
