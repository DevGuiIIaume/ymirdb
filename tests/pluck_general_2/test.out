> ok

> ok

> ok

> ok

> d [a b c]
c [b]
b [a]
a [1]

> a

> c

> index out of range

> b

> d []
c [b]
b [a]
a [1]

> nil

> nil

> nil

> b, c

> a

> nil

> a, b

> nil

> ok

> d [c b a]
c [b]
b [a]
a [1]

> nil

> a

> a, b

> a, b, c

> b, c, d

> c, d

> d

> nil

> bye
